### PR TITLE
GraphQL - Anime Mutations

### PR DESCRIPTION
**Pull request by toyhammered**
# Checklist

- [ ] All files pass Rubocop
- [ ] Any complex logic is commented
- [ ] Any new systems have thorough documentation
- [ ] Any user-facing changes are behind a feature flag (or: explain why they can't be)

- [ ] All the tests pass
- [ ] Tests have been added to cover the new code

### CODE REVIEW

**Comment by toyhammered** [comment]
dependent on #736 being merged

**Comment by timw4mail** [comment]
This is to update the anime type library entry?

**Comment by toyhammered** [comment]
> This is to update the anime type library entry?

No, this is to create, update, delete anime. #736 handles the updates of library entries.

**Review by NuckChorris — APPROVED** [review]

File: app/graphql/mutations/anime/create.rb [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
-    authorize Anime, :create?
-    anime = Anime.create!(input.to_model)
-    { anime: anime }
+  def load_anime(value)
+    ::Anime.new(value.to_h)
+  end
+
+  def authorized?(anime:)
+    super(anime, :create?)
+  end
```
Comment (NuckChorris) [inline app/graphql/mutations/anime/create.rb:16]: Retracting my approval on the basis of: I'm pretty sure this won't work

**Review by NuckChorris — CHANGES_REQUESTED** [review]

**Review by NuckChorris — APPROVED** [review]
After discussion, seems this will work through Magic™

**Pull request merged**